### ISSUE TITLE
Hello I need

### ISSUE THREAD

**Issue by qcj1206** [matyasbohacek/protecting-world-leaders-against-deepfakes#1]

**Comment by qcj1206** [comment]
Is the dataset for the Protecting World Leaders Against Deep Fakes? Thanks

**Comment by maty-bohacek** [comment]
Hi @qcj1206, you can download the data using the links in the csv files.

**Comment by qcj1206** [comment]
Yeah I know! thanks.I want to know these are the all dataset in paper in 2019 
Protecting World Leaders Against Deep Fakes
Shruti Agarwal and Hany Farid
University of California, Berkeley
Berkeley CA, USA
I need Hillary Clinton Barack Obama Bernie Sanders Donald Trump  Elizabeth Warren and so on!

**Comment by maty-bohacek** [comment]
No -- this database was released with [this PNAS paper](https://www.pnas.org/doi/10.1073/pnas.2216035119).

**Comment by qcj1206** [comment]
> 不，这个数据库是和[这篇PNAS论文](https://www.pnas.org/doi/10.1073/pnas.2216035119)一起发布的。

OK thank you again！
